### ISSUE TITLE
jsvu run failed:"SyntaxError: Unexpected token"

### ISSUE THREAD

**Issue by cxfeng1** [GoogleChromeLabs/jsvu#14]
```
➜  ~ jsvu
/usr/local/lib/node_modules/jsvu/cli.js:103
(async () => {
       ^
SyntaxError: Unexpected token (
    at Object.exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:513:28)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.runMain (module.js:575:10)
    at run (node.js:348:7)
    at startup (node.js:140:9)
    at node.js:463:3
```

My node version is `v6.2.2`

**Comment by cxfeng1** [comment]
It looks to be solved when I upgrade my node verstion to `v9.3.0`

**Comment by mathiasbynens** [comment]
What version were you using before? I guess it was older than v8.9.0: https://github.com/GoogleChromeLabs/jsvu#installation
